core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -327,6 +351,10 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
           .css('width', '24px')
           .css('padding', '0px 0px');
       });
+      $('.cke_inner').append(
+        '<span id="offline-warning">* Tools with dark background ' +
+        'can not be used when offline.<span>');
+      $('#offline-warning').css('display', 'none');
```

Comment:
Hey, can you not jquery here but instead use an ElementRef using ViewChild decorator. The issue with jquery is that it queries all elements in the dom with the class name and performs the changes to all of them.

This becomes an issue when you have two rte editors on a single page and they keep updating each other.

This comment applies for `.cke_inner`, not sure for `#offline-warning`

Reply:
Changed it.